Src/PerformanceCollector/Shared/Implementation/PerformanceCounterStructure.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    /// <summary>Represents basic performance counter structure.</summary>
```

Comment:
parsing of the full counter name into three pieces was made before based on PerformanceCounter class, now it should be more abstract from PerfCounter realization as we can have logical ones that are not mapped to the physical. also those objects were actually not used, but PerformanceCounter is disposable and nobody dealt with it correctly.